docs/source/services/timelock_service/index.rst

```diff
@@ -0,0 +1,36 @@
+.. _external-timelock-service:
+
+External Timelock Service
+=========================
+
+.. warning::
+
+   Running external timestamp and lock services is currently an experimental feature.
+   If you are interested in testing it, please contact the AtlasDB team.
+
+.. danger::
+
+   Improperly configuring one's cluster to use external timestamp and lock services can result in **SEVERE DATA
+   CORRUPTION**! Please contact the AtlasDB team if you wish to try this feature.
+
+.. toctree::
+    :maxdepth: 1
+    :titlesonly:
+
+    installation
+    server_configuration
+    client_configuration
+    migration
+
+The AtlasDB Timelock Service is an external implementation of the Timestamp and Lock services. Running an external
+Timelock Service (as opposed to running timestamp and lock services embedded within one's clients) provides several
+benefits:
+
+- Additional reliability; we rely on the `Atomix <http://atomix.io>`__ open-source distributed coordination library,
+  which has been Jepsen tested for fault tolerance.
+- To some extent, individual services can now scale independently of AtlasDB, depending on one's requirements.
```

Comment:
Yep, it means that you can have different numbers of clients and AtlasDB servers. This could be useful if you have large numbers of clients, since you have a fair bit of message complexity with all of the heartbeats going on if you maintained a distributed leader.